### PR TITLE
yj 5.0.0 (new formula)

### PR DESCRIPTION
**Pull request by sclevine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Requested here: https://github.com/sclevine/yj/issues/5

### CODE REVIEW

File: Formula/yj.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Yj < Formula
+  desc "CLI to convert between YAML, TOML, JSON, and HCL"
+  homepage "https://github.com/sclevine/yj"
+
+  url "https://github.com/sclevine/yj/archive/v5.0.0.tar.gz"
+  sha256 "df9a4f5b6d067842ea3da68ff92c374b98560dce1086337d39963a1346120574"
+  license "Apache-2.0"
+  head "https://github.com/sclevine/yj"
```
Comment (dawidd6) [inline Formula/yj.rb:8]:
```suggestion
  head "https://github.com/sclevine/yj.git"
```

File: Formula/yj.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Yj < Formula
+  desc "CLI to convert between YAML, TOML, JSON, and HCL"
+  homepage "https://github.com/sclevine/yj"
+
```
Comment (dawidd6) [inline Formula/yj.rb:4]:
```suggestion
```

File: Formula/yj.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class Yj < Formula
+  desc "CLI to convert between YAML, TOML, JSON, and HCL"
```
Comment (claui) [inline Formula/yj.rb:2]:
```suggestion
  desc "CLI to convert between YAML, TOML, JSON and HCL"
```
See our [Prose Style Guidelines](https://docs.brew.sh/Prose-Style-Guidelines#typographical-conventions).

File: Formula/yj.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  url "https://github.com/sclevine/yj/archive/v5.0.0.tar.gz"
+  sha256 "df9a4f5b6d067842ea3da68ff92c374b98560dce1086337d39963a1346120574"
+  license "Apache-2.0"
+  head "https://github.com/sclevine/yj"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"yj", "."
```
Comment (dawidd6) [inline Formula/yj.rb:13]:
```suggestion
    system "go", "build", *std_go_args
```

**Comment by sclevine** [comment]
Addressed the review comments and squashed the changes into the original commit.

**Comment by sclevine** [comment]
Addressed the review comments and squashed the changes into the original commit.

**Review by claui — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@sclevine, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by sclevine** [comment]
Apologies, I missed a build arg that's necessary for yj to report its version.
Opened https://github.com/Homebrew/homebrew-core/pull/58622 to correct this.

**Pull request closed without merging**